src/Pages/RoomDetail/BottomInfo/ResevationTab/Option/Option.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+import Theme from '../../../../../Styles/Theme';
+
+const Option = ({ option }) => {
+  const { image, name, price, discount_price } = option;
+
+  return (
+    <OptionWrapper key={name}>
```

Comment:
```suggestion
    <OptionWrapper>
```
map method 등을 사용하는 것이 아니기 때문에 불필요해보입니다